docs/architecture/security/principles/01-servers-are-not-trusted.mdx

```diff
@@ -0,0 +1,37 @@
+# P01 - Servers are not trusted
+
+Clients do not need to trust Bitwarden server or web infrastructure to be good actors in supplying
+data to keep an account's encrypted content secure. In other words, the special status of providing
+a network sync for client data does not enable a server, or any entity between the server and
+client, to take any action that would aid in decryption of a user's vault by the server.
+
+This is what we mean when we sometimes refer to "End-to-end encrypted."
+
+## Account key sharing as a feature
+
+This principle does not mean that clear text data are never shared, but rather that any such
+exposure is always declared to the user and exclusively between accounts, never to the server or
+infrastructure.
```

Comment:
suggestion: infrastructure is a bit vague here and has not been defined earlier in the text, I think we can just remove it

Reply:
If we do, I think we need a term to mean anything not the local client. We discuss this a bit above that the goal isn't to not trust the server repo code, but anything the client communicates to while pulling cloud-stored data.

I'm not yet sure how to say that, but it feels like a sticking point we need to handle. Maybe something as simple as cloud or web infrastructure?

Reply:
I suggested adding `Clients do not need to trust Bitwarden’s server or infrastructure` further up which I think would resolve this comment. I mostly reacted to `infrastructure` being mentioned for the first time in the middle of the text